src/styles/introjs.scss

```diff
@@ -320,11 +360,13 @@ tr.introjs-showElement {
 
 .introjs-bullets {
   text-align: center;
+  padding-top: 10px;
+  padding-bottom: 10px;
```

Comment:
Just a thought: wonder if this would be worth converting into a $spacing variable. Could be easier to manage/extend. Could use default too so it's easily overridden:

$spacing: 10px !default;

Reply:
good idea. I can work on it in a different PR.